CorsixTH/Lua/config_finder.lua

```diff
@@ -475,59 +474,53 @@ audio_music = nil -- [[X:\ThemeHospital\Music]]
+---------------------------------------------------------------------------------------------------
 -- Room information dialogs: Information about new rooms, important for
 -- additional rooms in later levels. Affects campaign only.
--- ]=] .. '\n' ..
+--]=] .. '\n' ..
 'room_information_dialogs = ' .. tostring(config_values.room_information_dialogs)  .. '\n' .. [=[
 
--------------------------------------------------------------------------------------------------------------------------
+---------------------------------------------------------------------------------------------------
 -- If true, parts of the hospital can be made inaccessible by blocking the path
 -- with rooms or objects. If false, all parts of the hospital must be kept
 -- accessible, the game will disallow any attempt to blocking the path.
--- ]=] .. '\n' ..
+--]=] .. '\n' ..
 'allow_blocking_off_areas = ' .. tostring(config_values.allow_blocking_off_areas) .. '\n' .. [=[
 
--------------------------------------------------------------------------------------------------------------------------
--- Direct Zoom: An experimental enhancement to avoid rendering to an
--- intermediate texture when zooming. Improves performance and reliability on
--- some hardware.
--- ]=] .. '\n' ..
+---------------------------------------------------------------------------------------------------
+-- Direct Zoom: Avoid rendering to an intermediate texture when zooming.
+-- Improves performance and reliability on some hardware.
+--]=] .. '\n' ..
 'direct_zoom = ' .. tostring(config_values.direct_zoom) .. '\n' .. [=[
 
--------------------------------------------------------------------------------------------------------------------------
-
------------------------------------------------- CAMPAIGN MENU -----------------------------------------------
--- By default your computer log in will be your name in the game.  You can change it in the
--- campaign menu or between the brace brackets below [[YOUR NAME]].
--- Note: space is limited in the game, so don't enter a name that is too long!
+---------------------------------------------------------------------------------------------------
+-- By default your username will be your name in the game. You can change it in
+-- the New Game menu or between the brace brackets below like [[NAME]].
+-- Note: space is limited in the game, so names will be cut off after 12 characters
 --
--- If you have specified any other locations for things like saves, music or screenshots you will find these below.  If
--- you change your mind and wish to go back to the default folders, just delete the relevant line.
---
--- If you wish to go back to the default settings for everything, you can delete this text file and it will be re-created
--- you play the game.
--------------------------------------------------------------------------------------------------------------------------
-
+player_name = [[]]
```

Comment:
This is going to wipe out player_name when it's saved again right? needs to be player_name = [[ .. config_values.player_name .. ']]\n'